larky/src/main/java/com/verygood/security/larky/modules/vgs/cerebro/text/analyzer/dto/TextPIIEntity.java

```diff
@@ -1,20 +1,19 @@
 package com.verygood.security.larky.modules.vgs.cerebro.text.analyzer.dto;
 
-import lombok.Builder;
+import com.verygood.security.larky.modules.types.PyProtocols;
+import com.verygood.security.larky.modules.types.structs.SimpleStruct;
 import net.starlark.java.annot.StarlarkBuiltin;
 import net.starlark.java.annot.StarlarkMethod;
 import net.starlark.java.eval.Dict;
 import net.starlark.java.eval.StarlarkFloat;
 import net.starlark.java.eval.StarlarkInt;
-import net.starlark.java.eval.StarlarkValue;
+import net.starlark.java.eval.StarlarkThread;
 
-
-@Builder
 @StarlarkBuiltin(
     name = "TextPIIEntity",
     category = "BUILTIN",
     doc = "The type represents TextPIIAnalyzer text analysis result, and describes an instance of PII entity.")
-public class TextPIIEntity implements StarlarkValue {
```

Comment:
https://github.com/verygoodsecurity/starlarky/blob/master/larky/src/main/java/com/verygood/security/larky/modules/types/LarkyObject.java#L49-L65

Did you want to define the `__name__` and `__class__` so you can get the right `repr()` and `str()`?